packages/router/src/std/router.ts

```diff
@@ -17,12 +17,11 @@ export const stdRouter = (
   services: ServicesInterface,
 ) => {
   (async function () {
-    const result = await typedMessageRouter(req.penumbraSwReq, services);
+    await typedMessageRouter(req.penumbraSwReq, services);
     sendResponse({
       sequence: req.sequence,
       penumbraSwRes: {
         type: req.penumbraSwReq.type,
-        data: result,
```

Comment:
I don't understand enough about the router to know if this removal is OK. If it is fine, then LGTM. But in the future, for changes like this, could you please add inline comments explaining these changes to make it easier on reviewers?

Reply:
i'll add PR comments in the future. i expected to pair on you with this. inline code comments feel out of place, these are vestigial remnants of the old and non-functional router